app/src/main/java/com/example/babytracking/PickerSetup.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.babytracking
+
+import android.app.DatePickerDialog
+import android.app.TimePickerDialog
+import android.content.Context
+import android.widget.TextView
+import com.example.babytracking.TimeFormatter
+import kotlinx.android.synthetic.main.sleep_layout.*
+import java.util.*
+
+/*
+This object allows the creation of date and time pickers.
+*/
+object PickerSetup {
+
+     fun setupDatePicker(context: Context, date_TextView: TextView, cal: Calendar) {
```

Comment:
You should not set a text view from a helper class. Can you return the string to show instead?